### DISCUSSION TITLE
Reload command

### DISCUSSION THREAD

**Discussion by joti366** [iterate-ch/cyberduck#16595] [Q&A]
I use Mountain Duck to do calculations in my local PyCharm with data from a server. Is there a way to reload the mounted drive from within the Python file (possibly via bash or AppleScript), just before the calculation? Right now I manually go to Finder and click Reload there before running the Python calculations. I already fiddled around in AppleScript and Automator, but couldn't make it work without Finder coming to the front. I would want the refresh to happen in the background. 

Thanks!

**Comment by dkocher** [comment]
I assume you managed to trigger the _Reload_ item in the Mountain Duck Finder context menu using AppleScript but this causes Finder.app to open ore come to foreground. Unfortunately there is currently no scripting interface in Mountain Duck that you could make use of.

If you have trouble that a file is read from cache when already updated on the server but not yet refreshed on the mounted volume you can try the following [hidden configuration option](https://docs.mountainduck.io/mountainduck/preferences/#hidden-configuration-options) to always read the latest file attributes from server and discard the cached file when a change is detected using the setting

```
fs.filenames.nocache.regexp=\\.*
```

The default setting is 

```
fs.filenames.nocache.regexp=^(?!~\\$|\\._).*\\.(docx|doc|docm|ppt|pptx|pptm|xls|xlsx|xlsm|vsd|vsdx|vsdm|odt|ods|odp|odf|odb|accdb|indd|ai|psd|prproj)
```

You may want to amend with a specific file extension only.
